### PR TITLE
Use S3 as storage backend

### PR DESCRIPTION
**Pull request by arnaud-morvan**
Upload and publish to S3

### CODE REVIEW

File: c2corg_images/views.py [hunk 1]
```diff
@@ -100,8 +102,25 @@ def upload(request):
...
+
     log.debug('%s - returning response', pre_key)
     return {'filename': pre_key + '.' + kind}
+
+
+@view_config(route_name='publish', renderer='json')
+def publish(request):
+    if request.POST['secret'] != os.environ['API_SECRET_KEY']:
+        raise HTTPForbidden('Bad secret key')
+    object = request.POST['object']
```
Comment (tsauerwein) [inline c2corg_images/views.py]: Is `object` a good name? Isn't this the image key?

Comment (arnaud-morvan) [inline c2corg_images/views.py]: yes, filename, object key

**Comment by gberaudo** [comment]
LGTM, please merge when @tsauerwein is addressed.

**Comment by arnaud-morvan** [comment]
Problem with travis search branch in c2corg repo instead of my fork.
Superseeded by #4

**Pull request closed without merging**